src/Layer/CopcLayer.js

```diff
@@ -45,15 +46,29 @@ class CopcLayer extends PointCloudLayer {
             this.spacing = source.info.spacing;
 
             this.root = new CopcNode(0, 0, 0, 0, pageOffset, pageLength, this, -1);
-            this.root.bbox.min.fromArray(cube, 0);
-            this.root.bbox.max.fromArray(cube, 3);
+
+            let forward = (x => x);
+            if (this.source.crs !== this.crs) {
+                try {
+                    forward = proj4(this.source.crs, this.crs).forward;
+                } catch (err) {
+                    throw new Error(`${err} is not defined in proj4`);
+                }
+            }
```

Comment:
Could be factorized in parent method

Reply:
done.
I added a new methode in PointCloudLayer.js to compute the bbox.

Reply:
ok